django/applications/catmaid/static/js/WindowMaker.js

```diff
@@ -130,6 +130,15 @@ var WindowMaker = new function()
    * configuration.
    */
   var createWidget = function(instance) {
+    try {
+      CATMAID.throwOnInsufficientWebGlContexts(instance.MIN_WEBGL_CONTEXTS);
```

Comment:
`instance.MIN_WEBGL_CONTEXTS` will be `undefined` for most widgets. While this happens to work with the test in `throwOnInsufficientWebGlContexts()`, I think it would be nicer if this would read `CATMAID.throwOnInsufficientWebGlContexts(instance.MIN_WEBGL_CONTEXTS || 0)` to make clear that widgets don't have to have this field.